src/main/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
Is this file required? If not, remove it from the commit. 
Delete the file in your local and then `git add -u` will add the deleted status of the file

Reply:
I will remove it.